src/Components/ExperimentOne/ExperimentOneUserForm.js

```diff
@@ -1,66 +1,87 @@
-import React, { useState } from 'react';
-import BoredAPICaller from '../../ApiTemplates/BoredAPICaller.js'
+import React, { useState } from "react";
+import BoredAPICaller from "../../ApiTemplates/BoredAPICaller.js";
 
- 
-function DisplayUserRecommendationBasedOnQuery(userActivityFromApi,userActivityTypeFromApi){
-    alert("Hey XYZ. Based on what you have told me, I would suggest doing something along the lines of " +
-    userActivityTypeFromApi + ". Therefore, maybe try "+userActivityFromApi.toLowerCase());
+function DisplayUserRecommendationBasedOnQuery(
+  userActivityFromApi,
+  userActivityTypeFromApi
+) {
+  alert(
+    "Hey XYZ. Based on what you have told me, I would suggest doing something along the lines of " +
+      userActivityTypeFromApi +
+      ". Therefore, maybe try " +
+      userActivityFromApi.toLowerCase()
+  );
 }
 
-function ExperimentOneUserForm(props){
+function ExperimentOneUserForm(props) {
+  const activityTypes = [
+    "social",
+    "recreational",
+    "busywork",
+    "diy",
+    "charity",
+    "cooking",
+    "relaxation",
+    "music",
+    "educational",
+  ];
+  const listOfParticipants = [1, 2, 3, 4, 5];
+  const [userName, setUserName] = useState("");
+  const [numberOfParticipants, setNumberOfParticipants] = useState("1");
```

Comment:
This guy should deffo be a number not a string based on the name

Reply:
Yup missed that one, thanks